tests/zenko_tests/node_tests/smoke_tests/vault_admin_and_IAM_API_tests/iamApi.js

```diff
@@ -0,0 +1,43 @@
+const async = require('async');
+
+const VaultClient = require('./VaultClient');
+
+const clientAdmin = VaultClient.getAdminClient();
+const accountName = 'iam-api-test-account';
+const accountInfo = {
+    email: `${accountName}@test.com`,
+    password: 'test',
+};
+const externalAccessKey = 'DZMMJUPWIUK8IWXRP0HQ';
+const externalSecretKey = 'iTuJdlidzrLipymvAGrLP66Yxghl4NQxLZR3cLlu';
+const userName = 'iam-api-test-user'
+
+describe('IAM users: ', () => {
+    let iamAccountClient = null;
+
+    beforeEach(done => async.series([
+        next => clientAdmin.createAccount(accountName, accountInfo, next),
+        next => clientAdmin.generateAccountAccessKey(accountName, next, { externalAccessKey, externalSecretKey }),
+    ], () => {
```

Comment:
Let's make sure we return an error if "before each" step fails. That will help us debug it.
```
if (err) {
   return done(err);
}
```

Reply:
Changed in review commit: [`b875a09` (#1207)](https://github.com/scality/Zenko/pull/1207/commits/b875a09c7268760a05fe9973830a9119b606b17b)